data/i18n/hi/hi.toml

```
@@ -1,6 +1,13 @@
 # i18n strings for Hindi version of the site.
 # NOTE: Please keep the entries in alphabetical order when editing
 
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
+[conjunction_1]
+other = "और"
+
 [caution]
 other = "सावधानी:"
```

Shouldn't this be सावधान? Since we're using this as a verb and not a noun?
```suggestion
other = "सावधान:"
```

---

Done!